### PR TITLE
Added method to get the determinant of two vectors.

### PR DESCRIPTION
**Pull request by ducklord**
This would be a nice util function to have in the Vec2 class.

### CODE REVIEW

**Comment by nanaze** [comment]
We can't take changes without signing the CLA

http://cla.corp.google.com/

File: closure/goog/math/vec2.js [hunk 1]
```diff
@@ -269,6 +269,16 @@ goog.math.Vec2.dot = function(a, b) {
   return a.x * b.x + a.y * b.y;
 };
 
+/**
+ * Returns the detminant of the two vectors.
+ * @param {!goog.math.Vec2} a The first vector.
+ * @param {!goog.math.Vec2} b The second vector.
+ * @return {number} The determinant of the two vectors.
+ */
+goog.math.Vec2.det = function(a, b) {
```
Comment (nanaze) [inline closure/goog/math/vec2.js]: full word please

File: closure/goog/math/vec2.js [hunk 2]
```diff
@@ -269,6 +269,16 @@ goog.math.Vec2.dot = function(a, b) {
   return a.x * b.x + a.y * b.y;
 };
 
+/**
+ * Returns the detminant of the two vectors.
```
Comment (nanaze) [inline closure/goog/math/vec2.js]: spelling

Comment (ducklord) [inline closure/goog/math/vec2.js]: Sorry about that.

File: closure/goog/math/vec2.js (hunk 1 above)
Comment (ducklord) [inline closure/goog/math/vec2.js]: The mathematical representation is "det", just like "lerp" is not written "linearInterpolation".

**Comment by concavelenz** [comment]
Confirmed signed CLA Feb 10, 2015

**Comment by joeltine** [comment]
Merged at SHA d07e083774f11d7dc4f84eba497d2d27f1407229

**Pull request closed without merging**